py-utils/src/utils/msg_bus/message_bus/config.py

```diff
@@ -0,0 +1,33 @@
+
+class Config(object):
+    def __init__(self):
+        pass
+
+
+class KafkaConfig(Config):
```

Comment:
For now, Conf class  from cortx-utils can be used. These values need to be supplied from config file.